test/src/e2e_vm_tests/test_programs/should_fail/contract_dependencies_conflicting_salt/contract_a/src/main.sw

```diff
@@ -0,0 +1,16 @@
+contract;
+
+use contract_b::CONTRACT_ID as CONTRACT_B_ID;
+use contract_c::CONTRACT_ID as CONTRACT_C_ID;
+
+abi MyContract {
+    fn test_function();
+}
+
+impl MyContract for Contract {
+    fn test_function() {
+        let CONTRACT_B = CONTRACT_B_ID;
+        let contract_b_id = ContractId::from(CONTRACT_B);
+        let contract_c_id = ContractId::from(contract_c::CONTRACT_ID);
+    }
+}
```

Comment:
It looks like this test could be reduced a little?

```rust
contract;

abi MyContract {
    fn test_function();
}

impl MyContract for Contract {
    fn test_function() {
        let contract_b_id = ContractId::from(contract_b::CONTRACT_ID);
        let contract_c_id = ContractId::from(contract_c::CONTRACT_ID);
    }
}
```

I guess it doesn't matter as we never get to the point of building this particular test.